relay-quotas/src/is_rate_limited.lua

```diff
@@ -28,24 +29,39 @@
 -- The result is a Lua table/array (Redis multi bulk reply) that specifies
 -- whether or not the item was *rejected* based on the provided limit.
 assert(#KEYS % 2 == 0, "there must be 2 keys per quota")
-assert(#ARGV % 3 == 0, "there must be 3 args per quota")
-assert(#KEYS / 2 == #ARGV / 3, "incorrect number of keys and arguments provided")
+assert(#ARGV % 4 == 0, "there must be 4 args per quota")
+assert(#KEYS / 2 == #ARGV / 4, "incorrect number of keys and arguments provided")
+
+-- parse the incoming string into boolean, returns `nil` if could not parse which is also considered `false`
+local function parse_boolean(v)
+    if v == 'true' then
+        return true
+    elseif v == 'false' then
+        return false
+    else
+        return nil
```

Comment:
why does this nil branch exist?  it's used directly in a boolean expression which i assume will be treated like false.  Might as well simplify to `if ... true else false`